CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+2.3.1
+=====
+
+* (improvement) Add method `integrateHttpBasicAuth()` for handling a list of Http Basic Auth Credentials
+* (deprecation) Deprecate `handleHttpBasicAuth` method
```

Comment:
You can also directly add an entry in `UPGRADE.md` for `2.x to 3.0`, that the method was removed (then).